src/pci_agent/__main__.py

```
@@ -1,454 +1,21 @@
+import uvicorn
 
-    def log_message(self, format: str, *args) -> None:
-        """Log HTTP requests."""
-        print(f"[Agent] {args[0]} {args[1]} {args[2]}")
+from pci_agent.api import create_app
+from pci_agent.config import AgentConfig
 
 
 def main() -> None:
     """Start the PCI Agent HTTP server."""
+    host = os.environ.get("HOST", "0.0.0.0")
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)